docs/authenticated-grpc.md

```diff
@@ -3,7 +3,6 @@
 Oasis Core nodes communicate between themselves over various protocols. One of
 those protocols is [gRPC] which is currently used for the following:
 
-* Compute nodes talking to storage nodes.
```

Comment:
This entire list is no longer true as we do not use gRPC for inter-node communication. We have been using a libp2p-based protocol for a long while now.